packages/stryker/src/utils/objectUtils.ts

```diff
@@ -88,4 +73,8 @@ export function setExitCode(n: number) {
 
 export function base64Decode(base64EncodedString: string) {
   return Buffer.from(base64EncodedString, 'base64').toString('utf8');
+}
+
+export function normalizeWhiteSpaces(str: string) {
```

Comment:
What does this function do?

Reply:
It consolidates multiple consecutive white spaces into a single space. I'll add a comment.